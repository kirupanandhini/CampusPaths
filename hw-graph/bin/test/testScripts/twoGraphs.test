# Checks that two graphs can be created and are independent

# Create a graph
CreateGraph graph1
CreateGraph graph2

# Add nodes to each graph
AddNode graph1 n1
AddNode graph1 n2
AddNode graph2 m1
AddNode graph2 m2

# Add edges to each graph
AddEdge graph1 n1 n2 e1
AddEdge graph2 m1 m2 e2

# Print the nodes in the graph and the outgoing edges from n1 and m1
ListNodes graph1
ListChildren graph1 n1
ListNodes graph2
ListChildren graph2 m1
