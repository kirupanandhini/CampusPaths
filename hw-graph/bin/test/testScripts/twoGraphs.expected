# Checks that two graphs can be created and are independent

# Create a graph
created graph graph1
created graph graph2

# Add nodes to each graph
added node n1 to graph1
added node n2 to graph1
added node m1 to graph2
added node m2 to graph2

# Add edges to each graph
added edge e1 from n1 to n2 in graph1
added edge e2 from m1 to m2 in graph2

# Print the nodes in the graph and the outgoing edges from n1 and m1
graph1 contains: n1 n2
the children of n1 in graph1 are: n2(e1)
graph2 contains: m1 m2
the children of m1 in graph2 are: m2(e2)